CMakeLists.txt

```diff
@@ -64,26 +64,34 @@ if (NOT DEFINED USE_PYTHON_CASADI)
   set(USE_PYTHON_CASADI TRUE)
 endif()
 
+# Use importlib to find the casadi path without importing it. This is useful
+# to find the path for the build-time dependency, not the run-time dependency.
 execute_process(
     COMMAND "${PYTHON_EXECUTABLE}" -c
-    "import casadi as _; print(_.__path__[0])"
+    "import importlib.util; print(importlib.util.find_spec('casadi').submodule_search_locations[0])"
     OUTPUT_VARIABLE CASADI_DIR
     OUTPUT_STRIP_TRAILING_WHITESPACE)
 
 if (CASADI_DIR)
   file(TO_CMAKE_PATH ${CASADI_DIR} CASADI_DIR)
-  message("Found python casadi path: ${CASADI_DIR}")
+  message("Found Python casadi path: ${CASADI_DIR}")
 endif()
 
 if(${USE_PYTHON_CASADI})
-  message("Trying to link against python casadi package")
+  message("Trying to link against Python casadi package")
   find_package(casadi CONFIG PATHS ${CASADI_DIR} REQUIRED)
 else()
-  message("Trying to link against any casadi package apart from the python one")
+  message("Trying to link against any casadi package apart from the Python one")
   set(CMAKE_IGNORE_PATH "${CASADI_DIR}/cmake")
   find_package(casadi CONFIG REQUIRED)
 endif()
 
+set_target_properties(
```

Comment:
Not entirely sure whether this is the correct method to append an RPATH entry since I am a novice at CMake, but it added the `casadi` path to the list of paths to be searched, so I guess it is correct